src/Forms/ConfirmedPasswordField.php

```diff
@@ -185,7 +248,16 @@ public function Field($properties = [])
                     $field->setAttribute($name, $value);
                 }
             }
-
+            if (str_ends_with($field->getName(), '[_Password]')) {
+                $validator = $this->getApplicableValidator();
+                if (is_a($validator, EntropyPasswordValidator::class)) {
+                    $minStrength = $this->getMinPasswordStrengthForEvaluation();
+                    if ($minStrength !== -1) {
```

Comment:
```suggestion
                $minStrength = $this->getMinPasswordStrengthForEvaluation();
                if ($minStrength >= 0) {
```
The logic is all in `getMinPasswordStrengthForEvaluation()` - we don't need an "applicable validator".

Reply:
I think we still need the check about what the PasswordValidator is, if we're on a non-member form, getRequireStrongPassword() was called, however PasswordValidator configured is to RulesPasswordValidator, then we don't want to be making AJAX requests to the strength endpoint